utils/src/tempfile.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::env::temp_dir;
+use std::path::Path;
+
+use vmm_sys_util::{errno::Result, tempfile};
+
+/// Wrapper over vmm_sys_util::tempfile::TempFile::new().
+/// This creates a temporary file inside $TMPDIR if set,
+/// otherwise inside `/tmp` if non-Android system.
+pub fn tempfile() -> Result<tempfile::TempFile> {
```

Comment:
I'm not sure if this should be called simply `tempfile()` or if something like `create_temfile()` would be better.

Reply:
I'd rather keep it simple. I admit that I am biased because of the API offered by `tempfile` crate from crates.io, which exposes the method with the same name for creating an "anonymous" temporary file. 

The method is documented and commented, so its usage should be pretty clear, while its name, although does not explicitly suggest creation of a tempfile, it surely implies it, because you can see it as a short form of `get_tempfile()`.